src/assignments/tech-jobs-mvc.rst

```diff
@@ -0,0 +1,443 @@
+and rewrite the data portion of the application to make a true, database-backed
+model. However, using the existing ``JobData`` class to provide some basic data
+functionality lets you focus on the views and controllers for now.
+
+Your Assignment
+---------------
+
+The list below provides a general overview of your assigned tasks. Specific
+details for each part appear in the following sections, so be sure to read them
+carefully as you solve each problem.
+
+#. Review Carly’s code in the ``JobData`` file as well as in the existing
+   controllers and views.
+#. As her next step, Carly created two basic templates to show lists of jobs
+   for each field of the data (``employer``, ``location``, etc.). You will need
+   to add some details to these views to allow users to select and display jobs
+   stored in the system.
+#. Finally, Carly started working on the search feature, but only got as far as
+   writing the code to display the search form. She’s handed the project to you
+   to finish the rest. You’ll add code to the controller and view to process
+   and display and search results.
+
+Getting Started
+----------------
+
+Set up a local copy of the project:
+
+#. Visit the `repository page <https://github.com/LaunchCodeEducation/TechJobsMVC>`__
+   for this project. Fork the repository to create a copy under your own GitHub
+   account.
+#. Open the project in Visual Studio.
+
+.. flagging this as a spot where opening a project in windows developed on mac might need extra instructions
+
+#. Launch the application so you can refer to both the code as well as the running app
+   as you complete the assignment.
+
+1) Review the Code
+-------------------
+
+.. admonition:: Tip
+
+   One essential programming skill that you will develop is the ability to read
+   and understand someone else's code. This assignment begins with you
+   practicing exactly that. Make sure you carefully examine the provided code
+   BEFORE you start changing things.
+
+   Trying to change a code sample before understanding how it works leads to
+   confusion, frustration, and a broken program. DO NOT SKIP the code review!
+
+Carly created an ASP.NET MVC application and filled in some features. She
+refactored ``JobData`` to generate a list of ``Job`` objects based on
+your TechJobsOO work. She added controllers and views for *Home*,
+*List*, and *Search* pages. ``JobData`` now also builds lists for the
+``Employer``, ``Location``, ``PositionType``, and ``CoreCompetency`` objects.
+
+The Model
+^^^^^^^^^
+
+The ``Models`` directory contains an abstract class definition for ``JobField``. The ``Employer``, 
+``Location``, ``PositionType``, and ``CoreCompetency`` classes all inherit from ``JobField``. 
+model classes for all of the job fields and for the ``Job`` class itself.  
+
+The Data
+^^^^^^^^
+
+The ``JobData`` class in the ``Data`` directory serves the same purpose as before---it reads data from
```

Comment:
I think it would be helpful to link back to where we first learned about this class. I am having to switch between a lot of different windows to figure out the instructions.